sphinx_gallery/tests/tinybuild/examples/plot_repr.py

```diff
@@ -7,8 +7,14 @@
 
 # sphinx_gallery_dummy_images=2
 class A:
+    """Test class with `_repr_html_` method."""
+
     def _repr_html_(self):
         return "<p><b>This should print<b></p>"
 
 
 A()
+
+import numpy as np
+
+np.arange(6)
```

Comment:
Why add this in this PR, perhaps better as a separate one?

Reply:
... might be the cause of this failure for example

https://dev.azure.com/sphinx-gallery/sphinx-gallery/_build/results?buildId=1451&view=logs&j=66ad8d10-cb69-57fe-8782-e0500ab4b7b6&t=b2a44cde-8fb7-5f37-5248-a7b5141501c3&l=264

Reply:
sorry leftover cruft